### ISSUE TITLE
[xchain-thorchain] Update cosmos-client to latest version 

### ISSUE THREAD

**Issue by veado** [xchainjs/xchainjs-lib#415]
**Note:** Since `xchain-thorchain` depends on `xchain-client` / `xchain-crypto` #422 + #421 need to be done before!

Depends on #398, too.

Please update other dependencies as well.

ncu :eyes:  https://www.npmjs.com/package/npm-check-updates

(status 01-09-2021)

```ts
ncu
Checking xchainjs-lib/packages/xchain-thorchain/package.json
[====================] 8/8 100%

 @types/big.js             ^6.0.0  →   ^6.1.1     
 @xchainjs/xchain-client  ^0.10.1  →  ^0.10.2     
 @xchainjs/xchain-crypto   ^0.2.4  →   ^0.2.5     
 @xchainjs/xchain-util     ^0.3.0  →   ^0.3.1     
 axios                    ^0.21.0  →  ^0.21.1     
 cosmos-client             0.39.2  →  0.42.14     
 nock                     ^13.0.5  →  ^13.1.3    
```

**Comment by veado** [comment]
Please update other dependencies as well.

ncu :eyes:  https://www.npmjs.com/package/npm-check-updates

(status 01-09-2021)

```ts
ncu
Checking xchainjs-lib/packages/xchain-thorchain/package.json
[====================] 8/8 100%

 @types/big.js             ^6.0.0  →   ^6.1.1     
 @xchainjs/xchain-client  ^0.10.1  →  ^0.10.2     
 @xchainjs/xchain-crypto   ^0.2.4  →   ^0.2.5     
 @xchainjs/xchain-util     ^0.3.0  →   ^0.3.1     
 axios                    ^0.21.0  →  ^0.21.1     
 cosmos-client             0.39.2  →  0.42.14     
 nock                     ^13.0.5  →  ^13.1.3    
```

**Comment by matikucharski** [comment]
Any update to this? Old cosmos-client is using sr25519 package that may result in `WebAssembly.Compile is disallowed on the main thread, if the buffer size is larger than 4KB` error on chromium.

**Comment by mikewyszinski** [comment]
completed in another card
